pom.xml

```diff
@@ -46,7 +46,7 @@
 	</modules>
 
     <properties>
-        <openMRSVersion>2.5.9-SNAPSHOT</openMRSVersion>
+        <openMRSVersion>2.5.11-SNAPSHOT</openMRSVersion>
```

Comment:
Nice catch that this was behind... do we want/need the latest snapshot, or should we stick with 2.5.10 (I do see that Ian has added one bump fix to 2.5.11-SNAPSHOT)

Reply:
Not sure.  I also saw his fix, and figured we were on the snapshot before, probably best to stay on snapshots in the maintenance line.